libs/search-widget/src/components/answer/Answer.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import { _ } from '../../core/i18n';
+  import Expander from '../../common/expander/Expander.svelte';
+  import Feedback from './Feedback.svelte';
+  import type { Chat, Search } from '@nuclia/core';
+  import { getSortedResults } from '../../core/stores/search.store';
+  import Tile from '../../tiles/Tile.svelte';
+  export let answer: Partial<Chat.Answer>;
+  export let rank = 0;
+
+  const sources = getSortedResults(answer.sources?.resources);
+</script>
+
+<div class="sw-answer">
+  <div class="text">{answer.text}</div>
+  {#if answer.sources}
+    <div class="feedback">
+      <Feedback {rank} />
+    </div>
+    <Expander>
+      <h3
+        class="title-xs"
+        slot="header">
+        {$_('answer.sources')}
+      </h3>
+      <div class="results">
+        {#each sources as result}
+          <Tile {result} />
```

Comment:
The sources tiles are displayed always as text tiles. Is this expected ?

Reply:
Not always but yes in many case (when the best paragraph is coming from the title).
Fixed.